src/Dev/SapphireTest.php

```diff
@@ -628,52 +628,64 @@ public function assertDOSContains($matches, $dataObjectSet)
             }
 
             // We couldn't find a match - assertion failed
-            $this->assertTrue(
+            static::assertTrue(
                 $matched,
                 "Failed asserting that the SS_List contains an item matching "
                 . var_export($match, true) . "\n\nIn the following SS_List:\n"
-                . $this->DOSSummaryForMatch($dataObjectSet, $match)
+                . static::ListSummaryForMatch($list, $match)
             );
         }
     }
+
+    /**
+     * @deprecated 4.0.0:5.0.0 Use assertListContains() instead
+     *
+     * @param $matches
+     * @param $dataObjectSet
+     */
+    public static function assertDOSContains($matches, $dataObjectSet) {
```

Comment:
Shall we leave the deprecated methods with the same type? (i.e. non-static). It makes the upgrade path slightly safer. :)